app/scripts/controllers/metametrics.js

```diff
@@ -113,7 +129,7 @@ export default class MetaMetricsController {
     const segmentApiCalls = initState?.segmentApiCalls || {};
 
     this.store = new ObservableStore({
-      participateInMetaMetrics: null,
+      participateInMetaMetrics: true,
```

Comment:
Why is this always true now?

Reply:
yeah should definitely still be null, otherwise they are automatically opted in by default and events tracked prior to opting in

Reply:
my fault, I fixed it, thx :)